### PR TITLE
Rename the project name (Build changes)

### PR DESCRIPTION
**Pull request by kowshikn**
*Description of changes:*

* Renamed the project only in `build` references. (Not touching release, code or readme)
* Manually merged bala's PR.

*Testing*

* Build works
```
❯ ./gradlew clean build          
=======================================
Elasticsearch Build Hamster says Hello!
  Gradle Version        : 6.5
  OS Info               : Mac OS X 10.15.7 (x86_64)
  JDK Version           : 14 (JDK)
  JAVA_HOME             : /usr/local/Cellar/openjdk/14.0.1/libexec/openjdk.jdk/Contents/Home
  Random Testing Seed   : F17A4D7C39A260F0
  In FIPS 140 mode      : false
=======================================

> Task :data-prepper-plugins:service-map-stateful:test
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by net.bytebuddy.dynamic.loading.ClassInjector$UsingReflection$Dispatcher$Resolved (file:/Users/kowshikn/.gradle/caches/modules-2/files-2.1/net.bytebuddy/byte-buddy/1.4.26/c56b0e90e3b6f9f117a0a0356674e86e86ba8652/byte-buddy-1.4.26.jar) to method java.lang.ClassLoader.findLoadedClass(java.lang.String)
WARNING: Please consider reporting this to the maintainers of net.bytebuddy.dynamic.loading.ClassInjector$UsingReflection$Dispatcher$Resolved
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 1m 5s
75 actionable tasks: 14 executed, 61 up-to-date

```

* Ran example app and it works.

* github workflow should catch any other failure.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/sink/TestSink.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.amazon.situp.model.annotations.SitupPlugin;
+import com.amazon.situp.model.record.Record;
+import com.amazon.situp.model.sink.Sink;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@SitupPlugin(name = "test-sink", type = PluginType.SINK)
```
Comment (erosas) [inline data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/sink/TestSink.java:14]: The annotation is still Situp

File: data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/source/TestSource.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import com.amazon.situp.model.record.Record;
+import com.amazon.situp.model.source.Source;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@SitupPlugin(name = "test-source", type = PluginType.SOURCE)
```
Comment (erosas) [inline data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/source/TestSource.java:15]: annotation name

File: data-prepper-plugins/common/src/main/java/com/amazon/situp/plugins/sink/FileSink.java [hunk 3]
```diff
@@ -15,15 +13,13 @@
 import java.nio.file.Paths;
 import java.util.Collection;
 
-import static com.google.common.base.Preconditions.checkNotNull;
 import static java.lang.String.format;
 
 @SitupPlugin(name = "file", type = PluginType.SINK)
 public class FileSink implements Sink<Record<String>> {
-    private static final Logger LOG = LoggerFactory.getLogger(FileSink.class);
```
Comment (yadavcbala) [inline data-prepper-plugins/common/src/main/java/com/amazon/situp/plugins/sink/FileSink.java:23]: Are these intentional changes ? these seem to be a problem of merge ?

File: data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/sink/TestSink.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package com.amazon.situp.plugins.sink;
```
Comment (yadavcbala) [inline data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/sink/TestSink.java:1]: think we can remove this completely - not used anymore

File: data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/source/TestSource.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
+package com.amazon.situp.plugins.source;
+
```
Comment (yadavcbala) [inline data-prepper-plugins/common/src/test/java/com/amazon/situp/plugins/source/TestSource.java:2]: this one too - we can remove this

**Review by erosas — APPROVED** [review]
Looks good, the PR is huge so do a follow up one for the annotation changes.

**Comment by kowshikn** [comment]
> Looks good, the PR is huge so do a follow up one for the annotation changes. 

Yes the plan is to do the rename in four steps.

1. Build references
2. Code references
3. Readme references
4. Release references

File: data-prepper-plugins/common/src/main/java/com/amazon/situp/plugins/sink/FileSink.java (hunk 3 above)
Comment (kowshikn) [reply data-prepper-plugins/common/src/main/java/com/amazon/situp/plugins/sink/FileSink.java:23]: Problem of merge.

**Review by yadavcbala — APPROVED** [review]
Discussed offline - Kowshik will address the comments when renaming the code

**Pull request merged**